src/modules/common/hooks/useFilter.js

```diff
@@ -42,6 +42,10 @@ export default function useFilter(defaultFilters = {}) {
     [filters]
   );
 
+  useEffect(() => {
+    setFilters(inputFilters);
```

Comment:
**Syncing** props and state is usually an anti-pattern. 
It is OK though to **initialise** the state but not trying to keep them the sync since it leads to multiple sources of truth. Which probably was why the props name initially was `defaultFilters` (`initialFilters` would be fine too).

We need to test this thoroughly.

Reply:
Dont know if it is related to the ani pattern but going to http://0.0.0.0:8080/#/wallet now crashes the app 
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/c0158ed2-377c-4f59-9d4f-d55ace8e8b1c)

Reply:
Yes I dint like this fix, probably we can discuss a better fix as we know the problem

Reply:
What i did was playing around with the filters on different pages then going to /wallet. Cannot replicate this though, so it might be working. But please also make such sanity check 🙃